modules/ai/module.php

```diff
@@ -110,6 +110,9 @@ public function __construct() {
 
 		if ( is_admin() ) {
 			add_action( 'wp_enqueue_media', [ $this, 'enqueue_ai_media_library' ] );
+			// Temp disable until feature will be ready
+			//add_action( 'admin_init', [ $this, 'enqueue_ai_woocommerce' ] );
```

Comment:
remove?

Reply:
this is the disable

Reply:
`// Temp disable until feature will be ready`